rest/config_manager.go

```diff
@@ -345,7 +345,12 @@ func (b *bootstrapContext) GetDatabaseConfigs(ctx context.Context, bucketName, g
 
 	var attempt int
 	for attempt = 1; attempt <= configFetchMaxRetryAttempts; attempt++ {
-		base.InfofCtx(ctx, base.KeyConfig, "GetDatabaseConfigs starting (attempt %d/%d)", attempt, configFetchMaxRetryAttempts)
+		msg := fmt.Sprintf("Checking for database config (attempt %d/%d)", attempt, configFetchMaxRetryAttempts)
+		if attempt == 1 {
+			base.TracefCtx(ctx, base.KeyConfig, msg)
```

Comment:
I think this might be better at debug.

We know it only happens once (per bucket), so isn't super verbose, and it's useful when finding out what buckets were being looked in under normal circumstances (no retries) - that can diagnose things like incorrect RBAC permissions limiting bucket visbility.

```suggestion
			base.DebugfCtx(ctx, base.KeyConfig, msg)
```